Week4/week 4.ipynb

```diff
@@ -0,0 +1,335 @@
+    "            col_name=input(\"Enter Your Choice Of Which Thing You Want To Update :\").lower()\n",
+    "            \n",
+    "            print(col_name)\n",
+    "            if col_name=='first name':\n",
+    "                old_fname=input(\"Enter Old First Name :\")\n",
+    "                new_fname=input(\"Enter New First Name :\")\n",
+    "                query =f\"\"\"update contact\n",
+    "                                    set fname='{new_fname}'\n",
+    "                                    where fname='{old_fname}' \"\"\"\n",
+    "            elif col_name=='last name':\n",
+    "                old_lname=input(\"Enter Old Last Name :\")\n",
+    "                new_lname=input(\"Enter New Last Name :\")\n",
+    "                query =f\"\"\"update contact\n",
+    "                                    set fname='{new_lname}'\n",
+    "                                    where fname='{old_lname}'  \"\"\"\n",
+    "            elif col_name=='contact no':\n",
+    "                old_contact=input(\"Enter Old Contact No :\")\n",
+    "                new_conatct=input(\"Enter New Contact No :\")\n",
+    "                query =f\"\"\"update contact\n",
+    "                                    set fname='{new_contact}'\n",
+    "                                    where fname='{old_contact}' \"\"\"\n",
+    "            curobj.execute(query)\n",
+    "            print()\n",
+    "            print(\"Record(s) Update Successfully\")\n",
+    "            print()\n",
+    "    except Error as e:\n",
+    "            print()\n",
+    "            print(e)\n",
+    "            print()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "32d9da78",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def search_contact():\n",
+    "    try:\n",
+    "                curobj.execute('select fname , lname from contact')\n",
+    "                All_Record = curobj.fetchall()\n",
+    "                print('First Name\\tLast Name')\n",
+    "                for i in All_Record:\n",
+    "                    print(f'{i[0]}\\t\\t{i[1]}')\n",
+    "    except Error as e:\n",
+    "                print(e)\n",
+    "                    \n",
+    "    try:\n",
+    "                Name = input('Enter Name : ')\n",
+    "                curobj.execute(f'''select fname , lname , contact from contact\n",
+    "                                                where fname = '{Name}' ''')\n",
+    "                Contact_No =curobj.fetchall()\n",
+    "                print()\n",
+    "                print(\"$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$\",end='\\n')\n",
+    "                print('First Name\\tLast Name\\tContact No')\n",
+    "                print(\"$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$\",end='\\n')\n",
+    "                for i in Contact_No:\n",
+    "                     print(f'{i[0]}\\t\\t{i[1]}\\t\\t{i[2]}')\n",
```

Comment:
whats \t used for?

Reply:
The \t in Python represents a space. It allows us to insert space or tab between strings in a code.